src/Command/Db/DbSizeCommand.php

```diff
@@ -319,11 +332,11 @@ private function mysqlTablesInNeedOfOptimizing() {
      * @return int Estimated usage in bytes.
      */
     private function getEstimatedUsage(HostInterface $host, array $database) {
-        if ($database['scheme'] === 'pgsql') {
-            return $this->getPgSqlUsage($host, $database);
+        switch($database['scheme']) {
+            case 'pgsql': return $this->getPgSqlUsage($host, $database); break;
+            case 'mongodb': return $this->getMongoDbUsage($host, $database); break;
+            default: $this->getMySqlUsage($host, $database);
         }
```

Comment:
Missing `return` in the default case (currently outputs 0% for mysql)

```suggestion
            case 'pgsql':
                return $this->getPgSqlUsage($host, $database);
            case 'mongodb':
                return $this->getMongoDbUsage($host, $database);
        }
        return $this->getMySqlUsage($host, $database);
```